src/kemal.cr

```diff
@@ -17,13 +20,12 @@ at_exit do
   config.add_handler Kemal::Handler::INSTANCE
   config.add_handler HTTP::StaticFileHandler.new("./public")
 
-  server = HTTP::Server.new("0.0.0.0", config.port, config.handlers)
```

Comment:
Yup... But I didn't do that because my this code hasn't been merged yet :speak_no_evil:.